package.json

```diff
@@ -74,19 +74,18 @@
     "karma-jasmine": "~2.0.1",
     "karma-jasmine-html-reporter": "^1.4.2",
     "karma-safari-launcher": "^1.0.0",
-    "ng-packagr": "^5.3.0",
+    "ng-packagr": "^9.0.0",
     "node-sass": "^4.12.0",
     "protractor": "~5.4.0",
-    "rxjs": "~6.5.2",
+    "rxjs": "~6.5.4",
     "scss-bundle": "^2.5.1",
     "ts-node": "~8.3.0",
-    "tsickle": "^0.36.0",
-    "tslib": "^1.10.0",
     "tslint": "~5.18.0",
-    "typescript": "~3.5.3",
-    "zone.js": "~0.9.1"
+    "typescript": "~3.7.5",
+    "zone.js": "~0.10.2"
   },
-  "peerDependencies": {
-    "@angular/common": ">= 5"
+  "peerDependencies": {},
```

Comment:
@earshinov do you know if this is still valid?

Reply:
Yes, I don't think that `@angular/common` is very useful in `peerDependencies` since angular-archwizard does not target any Angular versions specifically.

Reply:
I think the initial reason we added `@angular/common` to `peerDependencies` was to ensure that a compatible version of angular is used (i.e. a version greater than or equals to 5). If we remove this I'm not sure if we face any issues after the release.

Reply:
Maybe you are right. Let's preserve it in `peerDependencies` to take care of users who might try to add the latest angular-archwizard to a project based on an old version of Angular

Reply:
Should we retain `5` or should we increase it to some more current version? I'm actually not sure up to which version Angular 9 is compatible with.

Reply:
I would leave `5` and let users report if they encounter any problems. In theory Angular versions should be backward compatible (barring engine changes such as ViewEngine → Ivy)

Reply:
Done in 472e425